arkane/encorr/ae.py

```diff
@@ -134,6 +144,49 @@ def execute(self, output_file: str = None):
                 logging.warning('Could not write atom energies to database. Captured error:')
                 logging.warning(str(e))
 
+    def save_correlation_mat(self, path: str, labels: List[str] = None):
+        """
+        Save a visual representation of the parameter correlation matrix.
+
+        Args:
+            path: Path to save figure to.
+            labels: Parameter labels.
+        """
+        try:
+            import matplotlib.pyplot as plt
+        except ImportError:
+            return
+
+        if self.ae.correlation is None:
+            raise Exception('Fit AECs before saving correlation matrix!')
+
+        if labels is None:
+            labels = list(self.ae.atom_energies.keys())
+
+        fig, ax = plt.subplots()
+        ax.matshow(self.ae.correlation, cmap=plt.cm.PiYG)
+
+        # Superimpose values as text
+        for i in range(len(self.ae.correlation)):
+            for j in range(len(self.ae.correlation)):
+                c = self.ae.correlation[j, i]
+                ax.text(i, j, f'{c: .2f}', va='center', ha='center', fontsize=8)
+
+        # Save lims because they get changed when modifying labels
+        xlim = ax.get_xlim()
+        ylim = ax.get_ylim()
+
+        ax.set_xticks(list(range(len(self.ae.correlation))))
+        ax.set_yticks(list(range(len(self.ae.correlation))))
+        ax.set_xticklabels(labels, fontsize=14)
```

Comment:
make the font sizes passable as a dictionary, or don't set them (since they will be hard to make generally look correct)

Reply:
Added!